app/views/results/index.html.erb

```diff
@@ -14,7 +21,11 @@
           Filter the results
         </button>
         <div class="govuk-toggle__target" id="searchFilters" data-qa="filters">
-          <%= render partial: 'results/location' %>
+          <% if params[:l] == "3" %>
```

Comment:
For readability, it might be useful to wrap this in a helper method. It's hard to tell what '3' represents